src/app/layout/header/header.component.html

```diff
@@ -0,0 +1,24 @@
+<p>header works!</p>
+<nav class="navbar bg-primary">
+  <h1>
+    <!--    go from admin panel to user panel-->
+    <a routerLink="/">Angular Admin</a>
+  </h1>
+
+  <ul>
+    <li routerLinkActive="active">
+      <a [routerLink]="['/']">Главная</a>
+    </li>
+    <li routerLinkActive="active">
+      <a [routerLink]="['/books']">Книги</a>
```

Comment:
As you created a constant with paths, you can use them here

Reply:
Ok